### DISCUSSION TITLE
extraction of certain information impossible

### DISCUSSION THREAD

**Discussion by Larbo53** [jsvine/pdfplumber#584] [Ask for help with specific PDFs]
Hello,
I would like to extract the supplier's information (top left block: Bourgeois Frères, . (see attached image) from the attached pdf.
What is the solution?
Os : MacOs Bigsur 11.6
Python : 3.9.7
pdfplumber : 0.6.0

Thank you.

Sincerely
[BOURGEOISFacture 21053886.pdf](https://github.com/jsvine/pdfplumber/files/7888720/BOURGEOISFacture.21053886.pdf)
<img width="466" alt="Capture d’écran 2022-01-18 à 12 36 51" src="https://user-images.githubusercontent.com/87364492/149930976-f608c9cb-4544-4230-96cd-e334d7c9d77b.png">
.

**Comment by samkit-jain** [comment]
Hi @Larbo53 Appreciate your interest in the library. I am not able to extract any text from that top left portion you have shared. If you try selecting the text over there, you'll see it is not possible and hence, you won't be able to extract that text via pdfplumber as well. As an alternative, you can run the PDF through an OCR software and then extract the required text.

**Comment by mkl-public** [comment]
In the case at hand the block of supplier information is _not_ drawn using text drawing instructions. Instead it is drawn directly using filled paths of curves and lines. Regular text extractors (which look for text drawing instructions) won't recognize any text in that block.

**Reply by Larbo53** [reply]
Can you close this discussion?

Thanks a lot.
